client/a8c-for-agencies/sections/referrals/consolidated-view/index.tsx

```diff
@@ -12,13 +12,9 @@ const getConsolidatedData = ( referrals: Referral[] ) => {
 	};
 
 	referrals.forEach( ( referral ) => {
-		consolidatedData.allTimeCommissions += referral.commissions;
 		consolidatedData.pendingOrders += referral.statuses.filter(
 			( status ) => status === 'pending'
 		).length;
-		if ( referral.statuses.includes( 'pending' ) ) {
-			consolidatedData.pendingCommission += referral.commissions;
-		}
```

Comment:
Removed so that we don't see the typescript warning and this component is not used yet.